django/utils/repr.py

```diff
@@ -0,0 +1,26 @@
+import builtins
+import reprlib
+
+
+class DjangoRepr(reprlib.Repr):
+
+    def config(self, limit):
```

Comment:
I didn't want to override existing reprlib's constructor but taking a look at the default Repr constructor it's actually just setting attributes I will modify that and add limit argument to it instead of `config` method 👍